internal/backoff/backoff.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+package backoff
+
+import (
+	"context"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+)
+
+const (
+	Jitter     = 0.10
```

Comment:
Maybe add a comment here explaining what e.g. the unit of this is, or how it works?

Reply:
updated in next commit